### DISCUSSION TITLE
[question] Site name change

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#17456] [How to] [answered]
# 🐞 bug report

### ⁉️ Describe the bug

My site recently changed its domain from `didilan.net` to `idlan.net`. I updated `didilan.net` to `idlan.net` in any file and data in the database. Now when logging in, the system gives me the following error message:
![Screenshot 2024-05-10 120307](https://github.com/YetiForceCompany/YetiForceCRM/assets/10230144/ec23ea86-3058-4413-a5cc-a50b719c4bce)

How can I avoid this message (and potentially re-register the unchanged copy)?

### 🌍 Your Environment

| Environment                         | Version / Name |
| ----------------------------------- | -------------- |
| YetiForce                           | 6.5.0
| Web server (name and version)       | apache 2                |
| PHP                                 |   8.1             |

**Comment by rskrzypczak** [comment, marked as answer]
Log in as admin and proceed as usual when registering.
